Actuators Page Edit (#1907)
PX4/PX4-user_guide: en/config/actuators.md

![Actuator Outputs - Multicopter diagram](../../assets/config/actuators/qgc_actuators_mc_outputs.png)

Separate tabs are displayed for each output bus supported by the connected flight controller: PWM AUX (IO Board output), PWM MAIN (FMU Board output), UAVCAN.
Motors and actuators (which are referred to as "functions") can be assigned to any physical output on any of the available busses.

:::note
PWM AUX outputs are preferred over the PWM MAIN outputs for controlling motors (they have lower latency).
:::

PWM outputs are grouped based on the hardware timer groups.
Meaning all the outputs in one group must operate under the same protocol at the same rate (e.g. PWM signal at 400Hz for all the outputs in one group).

> Therefore it is not possible to map Servo and a Motor in the same output group, as they usually operate under a different rate.

Reviewer: ```suggestion
Therefore it is not possible to map Servo and a Motor in the same output group, as they usually operate at a different rate.
```

> On PWM AUX tab there are optionally CAP outputs that are generally used as the [camera capture/trigger input](../peripherals/camera.md#trigger-configuration). However you can map the CAP outputs as an output just like all the other output configurations.
> 
> Likewise, other AUX outputs can be mapped to be used as camera capture/triggering input.

Reviewer: ```suggestion
The PWM AUX tab has CAP outputs that are generally used as the [camera capture/trigger input](../peripherals/camera.md#trigger-configuration).
However you can map the CAP outputs to other output functions, and other AUX outputs can be used as camera capture/triggering input.
```
